### ISSUE TITLE
Release date for versions should follow ISO8601 format

### ISSUE THREAD

**Issue by Argocyte** [MultiMC/MultiMC5#3605]
System Information
-----------------------------
MultiMC version: 0.6.11-1430

Operating System: win32

Summary of the issue or suggestion:
----------------------------------------------
Minecraft versions show released date in the American date format. This is not internationally friendly and can lead to confusion.

The date format could be changed to big endian to allow international recognisability.

What should happen:
------------------------------
Use the date format ISO8601 YYYY-MM-DD 

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
Add instance > Released

Suspected cause:
---------------------------
Ignorance of the iso8601 international standard for date formatting.

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)
![image](https://user-images.githubusercontent.com/35248818/107879927-a75d5a80-6ed3-11eb-90ff-1e52554c7694.png)

Additional Info:
---------------------------

**Comment by jamierocks** [comment]
I wonder if it'd be possible to just use the appropriate format for the locale.

**Comment by Argocyte** [comment]
I suppose it could inherit from window's date format settings?

**Comment by peterix** [comment]
Let's see...

It should be whatever is the date format on your system, depending on your configuration?

**Comment by Argocyte** [comment]
> Hmm, nevermind. It's actually based on the language you select in the settings.

Given the only English option is American English...

**Comment by peterix** [comment]
Yeah ... I guess that's fair enough.

The other option would be to allow setting this sort of stuff independently from the language.

**Comment by Argocyte** [comment]
The ISO format is international, and the standard for file naming and release schemes (sorting by alphabetical puts them in date order).

Given the usage of the date here ISO8601 seems highly appropriate as the default for all languages.

**Comment by peterix** [comment]
Yeah, I guess that's reasonable. We got timestamps in more than one place though, and this should probably be something that's configurable.

**Comment by peterix** [comment]
Anyway, we got translations, but that doesn't cover every language or locale possible.

**Comment by jamierocks** [comment]
> Hmm, nevermind. It's actually based on the language you select in the settings.

I guess that makes sense, annoying in this case though.

> The other option would be to allow setting this sort of stuff independently from the language.

That might be a good idea, or just add in more English's (English English, whatever others).

**Comment by Liggliluff** [comment]
If possible, it should use the format as provided by the system. That way, the user gets the format they expect and are used to, and you get a consistent user experience across the system. I've set my system to ISO 8601 because I think it's superior. But for dynamic date fields like these, it should base it on your system format first.

If the software can't access the system date format; then either offer all locales from CLDR to cover as much as possible, or just use the ISO 8601 format.

ISO 8601 is better used for static data, such as databases, since static data can't be adjusted for every locale; plus that ISO 8601 is easier for a computer to read, which is useful in databases. If a field is dynamic, it shouldn't force ISO 8601 on everyone, that should only be the last resort.

**Comment by kb-1000** [comment]
In my opinion, changing MultiMC ISO 8601 globally for all users is a bad idea. (For a datetime value even more, but this is just date.)
That doesn't seem to be a user-facing format. You might like it, but not everyone does.
Making it use the system date format seems like a good idea, if possible. That would also catch your ISO 8601 special case.

**Comment by MrMelon54** [comment]
I think letting the user pick their own date format in settings would be the best resolution for this. I don't want to be stuck using MM/DD/YYYY or YYYY-MM-DD when I'd prefer to use DD/MM/YYYY or just write out the full/shortened name of the month (e.g. 8 Apr 2021).

**Comment by jamierocks** [comment]
I agree. GH-3715 will allow you to have that date format with British English fwiw :)

**Comment by Liggliluff** [comment]
@MrMelon54 The actual solution is for the software to use the system format. If you prefer DD/MM/YYYY then you should set your system to that format, then all proper software will use it.

Adding an option in the setting is a roundabout way to patch an issue that still isn't resolved. It's fine to have an option to override the format for individual software. But the actual solution is for the software to use the system format, and for you to set the system format to your preferred format.

**Comment by MrMelon54** [comment]
Are there any American spellings in en_US which would be different for en_GB. It would be nice if those are also corrected.

I haven't noticed any American only spelling yet but I tend to just glance over any text as long as I understand what it means.

**Comment by jamierocks** [comment]
Yep, they should be amended - see https://translate.multimc.org/.

**Comment by AverageComet250** [comment]
I think there should be a config option in MMC for this, however it should default to the system format if the user hasn't gone and changed it
